cost/kubecost/cluster/kubecost_cluster_rightsizing_recommendations.pt

```diff
@@ -108,6 +108,73 @@ datasource "ds_kubecost_currency_code" do
   end
 end
 
+###############################################################################
```

Comment:
These style of headers are intended for indicating sections of the policy template, such as "Datasources & Scripts", "Parameters", etc. They shouldn't be used in other ways.

A normal comment should be used here instead.

Reply:
done!